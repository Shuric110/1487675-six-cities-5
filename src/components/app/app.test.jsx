import React from "react";
import renderer from "react-test-renderer";
import App from "./app";

jest.mock(`../main-screen/main-screen`, () => `MainScreen`);

describe(`Render App`, () => {
  it(`App for root path renders correctly`, () => {
    const tree = renderer
      .create(
          <App />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

});
