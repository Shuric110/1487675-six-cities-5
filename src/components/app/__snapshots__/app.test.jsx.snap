// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App App for root path renders correctly 1`] = `<MainScreen />`;
